sdcm/nemesis.py

```diff
@@ -4013,6 +4041,14 @@ def disrupt_grow_shrink_cluster(self):
         self._grow_cluster(rack=None)
         self._shrink_cluster(rack=None)
 
+    def disrupt_grow_shrink_cluster_parallel(self):
```

Comment:
Later we might want to use the same nemesis 

But have it configurable based on scylla capabilities and SCT configuration